app/src/main/java/com/disfluency/screens/pacientes/inputs.kt

```diff
@@ -0,0 +1,154 @@
+package com.disfluency.screens.pacientes
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CalendarToday
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.util.Collections
+data class Input<T>(val value: T, val wrongValue: ()->Boolean, val validate: (T)->Unit){
+    fun validate() {
+        validate(value)
+    }
+}
+
+@Composable
+fun inputString(label: String, validations: List<(String)->Boolean> = Collections.emptyList(), keyboardOptions: KeyboardOptions = KeyboardOptions.Default): Input<String>{
+    var value by remember { mutableStateOf("") }
```

Comment:
Aca estuve mirando y entiendo que esto no se debe usar asi, la idea es que el estado se cree mas arriba y se lo pase por parametro al input para que se llene, los composables entiendo no deben devolver estados sino modificarlos. 

Aca estuve leyendo dos posts con ejemplos de input
https://medium.com/@manojbhadane/android-login-screen-using-jetpack-compose-part-1-57ecfc74e428
Este un cacho mas complejo pero explica banda tambien
https://medium.com/androiddevelopers/effective-state-management-for-textfield-in-compose-d6e5b070fbe5